src/encoded/types/document.py

```diff
@@ -16,4 +16,5 @@
         'description': 'Listing of Documents',
     })
 class Document(SMAHTItem, CoreDocument):
```

Comment:
I wonder if we should just use a class like SmahtItem, though I guess that's not a whole lot more readable. Maybe ignore me...